solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/ModelQuery.java

```diff
@@ -75,11 +75,14 @@
   private Semaphore querySemaphore; // limits the number of threads per query, so that multiple requests can be serviced simultaneously
 
   public ModelQuery(LTRScoringModel meta) {
-    this(meta, false);
+    this(meta, null, false);
```

Comment:
The previous code already had efi (un)initialized to null.  The existing code checks for null efis, so this is consistent with the previous behavior.